packages/auth/package.json

```diff
@@ -61,6 +60,7 @@
     "typescript": "^2.8.1"
   },
   "dependencies": {
+    "bluebird": "^3.5.1",
```

Comment:
Typescript comes already with built-in promises, why rely on a 3rd party module?

Reply:
@josemigallas This is designed to work with the web and needs polyfil. Tried typescript promises and they are not polyfilled.